Cargo.toml

```diff
@@ -1,3 +1,22 @@
+[workspace]
+resolver = "2"
+members = [
+  "crates/cargo-platform",
+  "crates/cargo-test-macro",
+  "crates/cargo-test-support",
+  "crates/cargo-util",
+  "crates/crates-io",
+  "crates/credential/*",
+  "crates/home",
+  "crates/mdman",
```

Comment:
`"crates/*",`?

Reply:
No. That assumes `crates/credential/Cargo.toml` existing. See <https://github.com/rust-lang/cargo/issues/11405>.

Reply:
This is also addressable
- We could flatten our `crates` folder.  I know [matklad recommends it](https://matklad.github.io/2021/08/22/large-rust-workspaces.html) and I'm fine enough with it.
- We could make a `credentials/` top-level directory

Yes, it involves changing how we do things but being able to automatically scale up our processes I think is worth the compromises.